src/multi-input/index.jsx

```diff
@@ -1,48 +1,62 @@
+import { v4 as uuid } from 'uuid';
 import React, { useState, useEffect } from 'react';
 import { Button, Input } from '@ukhomeoffice/react-components';
+import castArray from 'lodash/castArray';
 
-function Item({ value, onRemove, showRemove, onChange }) {
+function Item({ item, onRemove, showRemove, onChange, name, disabled }) {
+  const isDisabled = disabled.includes(item.id);
```

Comment:
There doesn't seem to be anywhere that `disabled` is defaulted to a type with an `includes` function, so will this break on pre-existing implementations where `disabled` is undefined?